packages/wdio-utils/src/duration.ts

```diff
@@ -0,0 +1,44 @@
+class DurationTracker {
+    private timers: Record<string, number> = {}
+    private durations: Record<string, number> = {}
+
+    start(phase: string): void {
```

Comment:
Similar here and in `end`.

Reply:
I think it makes sense to check if a phase has been started before and if so throw an error. We don't want to start a phase multiple times and get skewed results.

Reply:
I agree! I've added validation to prevent duplicate phase starts, which could corrupt timing data.